packages/web/index.js

```diff
@@ -38,7 +38,6 @@ module.exports = (neutrino, opts = {}) => {
       hot: opts.hot !== false
     },
     style: {},
-    manifest: opts.html === false ? {} : false,
```

Comment:
Should we now throw if a `manifest` option is passed? (Like occurs for the other removed options in this file)

Reply:
👍